receiver/oracledbreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,29 @@
+receivers:
+  oracledb:
+    # driver name: oracle
+    # username: otel
+    # password: password
+    # host: localhost
+    # container exposed port: 51521
+    # Oracle DB service name: XE
+    # Refer to Oracle Go Driver go_ora documentation for full connection string options
+    datasource: "oracle://otel:password@localhost:51521/XE"
+processors:
+exporters:
+  file:
+    path: /tmp/out/file.txt
+  signalfx:
+    realm: us0
+    access_token: <token>
+service:
+  telemetry:
+    logs:
+      level: "debug"
+  pipelines:
+    metrics:
+      receivers:
+        - oracledb
+      processors:
+      exporters:
+        #- file
+        - signalfx
```

Comment:
We recently migrated all the config tests to use `confmap/confmaptest`, so files like this only include relevant parts. Can you please apply the same approach here? https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14749 can be used as reference